server/lib/sanitize-html.ts

```diff
@@ -89,6 +89,14 @@ export const buildSanitizerOptions = (allowedContent: AllowedContentType = {}):
     transformTags: {
       h1: 'h3',
       h2: 'h3',
+      a: function (tagName, attribs) {
+        const newAttribs = attribs;
+        newAttribs.href = `/redirect?url=${encodeURIComponent(attribs.href)}`;
+        return {
+          tagName: 'a',
+          attribs: newAttribs,
+        };
```

Comment:
Using a different variable (`newAttribs`) has no effect, as mutating `newAttribs` will still mutate `attribs`. So we can simplify (and make sure we don't mutate) like that:

```suggestion
        return {
          tagName: 'a',
          attribs: {
            ...attribs,
            href: `/redirect?url=${encodeURIComponent(attribs.href)}`,
          },
        };
```